server/views/pages/prisons/sessionTemplates/addSessionTemplate.njk

```diff
@@ -232,6 +232,43 @@
           ]
         }) }}
 
+        {{ govukRadios({
+          name: "visitOrderRestriction",
+          fieldset: {
+            legend: {
+              text: "What type of visiting order is used for this session?",
+              classes: "govuk-fieldset__legend--m"
+            },
+            attributes: { "data-test": "visit-order-restriction" }
+          },
+          items: [
+            {
+              value: "VO_PVO",
+              text: "VO or PVO",
+              checked: true
+            },
+            {
+              value: "VO",
+              text: "VO only",
+              checked: formValues.visitOrderRestriction === "VO"
+            },
+            {
+              value: "PVO",
+              text: "PVO only",
+              checked: formValues.visitOrderRestriction === "PVO"
+            },
+            {
+              divider: "or"
```

Comment:
Realised having the 'divider' option makes iterating over a constant to generate these radios a bit tricker. Two ways you could handle this:

1. within the loop, just do a check before adding the radio to the array, e.g.
```
{% if loop.last %}
  // push { divider: "or" } to array of items
{% endif %
```

2. keep with the full list of radios in the template as you have, but take the values from a constants file (so you get consistency of the labels without duplication)